packages/desktop/views/dashboard/governance/components/proposal-details/ProposalDetailsPane.svelte

```diff
@@ -1,21 +1,58 @@
 <script lang="ts">
-    import { MarkdownBlock, Text } from '@bloomwalletio/ui'
-    import { Pane } from '@ui'
+    import { Button, MarkdownBlock, Text } from '@bloomwalletio/ui'
     import { ProposalDetailsMenu, ProposalStatusPill } from '../'
     import { IProposal } from '@contexts/governance'
+    import { PopupId, openPopup } from '@desktop/auxiliary/popup'
+
+    const TEXT_LENGTH_TO_TRUNCATE = 40
 
     export let proposal: IProposal
+
+    function onReadMoreClick(): void {
+        openPopup({
+            id: PopupId.MarkdownBlock,
+            props: {
+                title: proposal?.title ?? '',
+                markdown: proposal?.additionalInfo,
+            },
+            hideClose: true,
+        })
+    }
 </script>
 
-<Pane classes="p-6 flex flex-col h-fit">
-    <header-container class="flex justify-between items-center mb-4">
-        <ProposalStatusPill {proposal} />
+<div class="flex flex-col h-fit">
+    <header-container class="flex justify-between items-start mb-4">
+        <Text type="h5">{proposal?.title}</Text>
         <ProposalDetailsMenu {proposal} modalPosition={{ right: '24px', top: '54px' }} />
     </header-container>
-    <div class="flex flex-1 flex-col space-y-4 justify-between scrollable-y">
-        <Text type="h4">{proposal?.title}</Text>
+    <div class="flex flex-1 flex-col space-y-4 justify-between">
+        <div class="flex">
+            <ProposalStatusPill {proposal} />
+        </div>
         {#if proposal?.additionalInfo}
-            <MarkdownBlock text={proposal?.additionalInfo} />
+            {@const truncate = proposal?.additionalInfo.length >= TEXT_LENGTH_TO_TRUNCATE}
```

Comment:
lets make sure `proposal` is not undefined. Maybe by wrapping the `ProposalDetailsView` with a `{#if $selectedProposal}`